make_tag.py

```diff
@@ -135,9 +135,9 @@ def make_tag(remote, bump_major, bump_minor, need_verification, github_actor):
             print('Skipped creating the tag')
             return
 
-    if github_token:
+    if github_actor:
```

Comment:
This is incorrect. We do not want to use Github APIs - but rather set the git actor on the regular git commands

Reply:
Nvm. Saw this PR - https://github.com/linkedin/venice/pull/64/files